### ISSUE TITLE
Microsoft.Extensions.Logging.EventLog not compitable with .NetCoreApp v1.0

### ISSUE THREAD

**Issue by karelz** [aspnet/Home#1952]
_From @zjleaman on March 6, 2017 20:13_

```
Package Microsoft.Extensions.Logging.EventLog 1.1.1 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package Microsoft.Extensions.Logging.EventLog 1.1.1 supports: net451 (.NETFramework,Version=v4.5.1)
    One or more packages are incompatible with .NETCoreApp,Version=v1.0.
```

Is there an alternative to writing to the Windows EventLog for a .NetCore app?
The EventLog nuget package doesn't install and I want to use it's LoggerProvider.

_Copied from original issue: dotnet/corefx#16785_

**Comment by natemcmaster** [comment]
Per discussion on https://github.com/dotnet/corefx/issues/6913 and https://github.com/dotnet/corefx/issues/16785, this is not possible even in .NET Core 2.0. We would need System.Diagnostic.EventLog ported to .NET Core or available as an extension to .NET Standard 2.0.

**Comment by Max-Riley** [comment]
This has been achieved via https://github.com/dotnet/corefx/pull/24344 and will be available in Core 2.1
